src/util/configInheritance.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { getConfigRowMetadata } from "util/configInheritance";
+import { StoragePoolFormValues } from "pages/storage/forms/StoragePoolForm";
+import { StorageVolumeFormValues } from "pages/storage/forms/StorageVolumeForm";
+
+beforeEach(() => {
+  vi.mock("@tanstack/react-query", () => ({
```

Comment:
where does `vi` come from? I assume vitest?

Reply:
Yes, this is from vitest and is quite nice. One oddity is you can't reference any variables in the mock code, because of hoisting. Need to remember that.

Reply:
oh that's a interesting one, good to know!